Modules/arraymodule.c

```diff
@@ -868,6 +868,20 @@ array_slice(arrayobject *a, Py_ssize_t ilow, Py_ssize_t ihigh)
     return (PyObject *)np;
 }
 
+/*[clinic input]
```

Comment:
This will also need new unit tests (probably in `test_array.py` but I haven't checked) and a documentation entry (in `array.rst`)

Reply:
> This will also need new unit tests (probably in `test_array.py` but I haven't checked) and a documentation entry (in `array.rst`)

Thanks for the comment. I'll try to do it now.

Reply:
done